### DISCUSSION TITLE
How to keep profile list closed by default

### DISCUSSION THREAD

**Discussion by ghost** [Eugeny/tabby#6065] [Q&A]
Hi,
I have alot of hosts grouped in different profiles, so my profile list is very long and each profile is by default developed:

For example:
![image](https://user-images.githubusercontent.com/81583999/161139287-1128fcda-2a71-4dee-ab51-9b7d00eec864.png)

Is there a way to have them stay closed by default like below?
![image](https://user-images.githubusercontent.com/81583999/161139388-16de396b-570c-4f62-9504-9d101ec3f372.png)

It's a small details but helps me a lot to have it.

Thank you again

**Comment by Eugeny** [comment]
The nightly build now remembers the collapsed state of the groups.

**Reply by ghost** [reply]
@Eugeny thanks, do you mean the 1.0.174?

**Reply by Eugeny** [reply]
No - I mean the builds that you can download here: https://nightly.link/Eugeny/tabby/workflows/build/master

**Reply by ghost** [reply]
@Eugeny thanks! I installed the windows x86_64 installer, but it seems not opening up....

**Reply by ghost** [reply]
I just tried portable version, it seems the same issue.

**Reply by ghost** [reply]
I think this is the issue report
[https://github.com/Eugeny/tabby/issues/6081]
